backend/api/Database/Models/Inspection.cs

```diff
@@ -44,7 +44,7 @@ public Inspection(CustomInspectionQuery inspectionQuery)
         public Inspection(Inspection copy, InspectionStatus? inspectionStatus = null, bool useEmptyIDs = false)
```

Comment:
```suggestion
        public Inspection(Inspection copy, InspectionStatus? inspectionStatus = null, bool useEmptyID = false)
```
Very pedantic sorry 😬

Reply:
Also, have you tested that this (not using an empty IsarTaskId) does not affect the MissionTask hash generation code?